cli/createTemplate.sh

```diff
@@ -53,22 +56,21 @@ GENERATION_FRAMEWORK    = "${GENERATION_FRAMEWORK_DEFAULT}"
 GENERATION_INPUT_SOURCE = "${GENRATION_INPUT_SOURCE_DEFAULT}"
 DISABLE_OUTPUT_CLEANUP  = "${DISABLE_OUTPUT_CLEANUP_DEFAULT}"
 ENTRANCE                = "${ENTRANCE_DEFAULT}"
+FLOWS                   = "${FLOWS_DEFAULT}"
 
 NOTES:
 
 1. You must be in the directory specific to the level
-2. DEPLOYMENT_UNIT must be one of "s3", "cert", "roles", "apigateway" or "waf" for the "account" level
-3. For the "segment" level the "baseline" unit must be deployed before any other unit
-4. When deploying network level components in the "segment" level you must deploy vpc before igw, nat, or vpcendpoint
 
 EOF
 }
 
 function options() {
 
   # Parse options
-  while getopts ":c:d:e:f:g:hi:l:o:p:q:r:u:xz:" option; do
+  while getopts ":b:c:d:e:f:g:hi:l:o:p:q:r:u:xz:" option; do
       case "${option}" in
+          b) FLOWS+=("${OPTARG}") ;;
```

Comment:
Need to initialise before getopts?

Reply:
This is setup on Line 19 during the setup process